packages/design-system-docs/src/pages/components/Autocomplete/Autocomplete.example.jsx

```diff
@@ -76,6 +76,56 @@ ReactDOM.render(
       />
     </Autocomplete>
 
+    <Autocomplete
+      items={[
+        {
+          id: '1',
+          name: 'Carrots (1)',
+          children: (
```

Comment:
Thinking about naming here - what about something like `autoCompleteAll` `autoCompleteGroup`?

Reply:
`item.children` is for providing React elements for individual results rather than leaving the autocomplete component in charge of rendering that item based on `item.name`. Are you talking about replacing the `item.children` with `item.autoCompleteAll/Group`? I imagined alternatives like `item.element` or `item.content`.